pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "6.4.0"
+	bundleVersion        = "6.6.1"
```

Comment:
Why does the version change here?

Reply:
I deployed this on gauss for testing but since there are already several other versions of the AWS-operator running there I had to change to a unique version number to prevent conflicts.